README.md

```diff
@@ -84,25 +90,33 @@ Second, the user has two options:
   option, or provide an arbitrary violation of parallel trends via the `delta()` option. 
 
 Let's illustrate the first use case:
+
 ```stata
-pretrends power 0.5, numpre(3) b(beta) v(sigma)
-* Slope for 50% power =  .0520662
+pretrends power 0.5, pre(1/3) post(4/7)
+
+* Slope for 50% power =  .0494932
 
 return list
 * scalars:
-*               r(slope) =  .0520662478209672
+*               r(slope) =  .0494932155202119
 *               r(Power) =  .5
 ```
 
-To visualize the linear trend against which pre-tests have 50 percent power:
+In the command above, the option `pre(1/3)` tells the package that the pre-treatment event-study coefficients are in positions 1 through 3 in our regression results. (The package assumes that the period before the event-study is normalized to zero and omitted from the regression.) Likewise, the option `post(1/4)` tells the package that the post-treatment coefficients are in positions 4 through 7. The results of the command tells us that if there was a linear violation of parallel trends with slope 0.049, then we would have 50% power to detect it (where we say it's "detected" if there is a significant pre-trend coefficient). If we want wanted a different power threshold, say 80%, we would change `power 0.5` to `power 0.8` in the command above.  
+
+
+Next, we illustrate how to visualize violations of parallel trends using the package's second subcommand. For simplicitly, lets visualize the linear trend against which pre-tests have 50 percent power that we just calculated. This is just for illustration --- you can visualize any violation that you want, and should choose an economically relevant one. To do this, we run the command:
```

Comment:
I believe `---` is replaced with a horizontal line, rather than an em-dash. You can insert an em-dash directly, —, or via HTML, `&mdash;`.